src/drive/web/modules/views/Folder/FolderViewBody.jsx

```diff
@@ -71,25 +72,25 @@ const FolderViewBody = ({
           {isEmpty && <EmptyDrive canUpload={true} />}
           {hasDataToShow && (
             <div className={isMobileApp() ? 'u-ov-hidden' : ''}>
-              {queryResults.map(query => (
-                <>
+              {queryResults.map((query, queryIndex) => (
+                <React.Fragment key={queryIndex}>
                   {query.data.map(file => (
                     <File
                       key={file._id}
                       attributes={file}
                       displayedFolder={null}
-                      onFolderOpen={navigateToFolder}
-                      onFileOpen={navigateToFile}
                       withSelectionCheckbox={true}
-                      withFilePath={false}
                       withSharedBadge={true}
                       isFlatDomain={true}
+                      onFolderOpen={navigateToFolder}
+                      onFileOpen={navigateToFile}
+                      withFilePath={withFilePath}
```

Comment:
Nit: Previously, props were almost sorted an this is not the case anymore. I think it's good when similar props are grouped, especially when there are a lot of them like here.